### DISCUSSION TITLE
No loop option

### DISCUSSION THREAD

**Discussion by isle9** [KRTirtho/spotube#446] [Q&A]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Pain

I'd like to have an option for no queue loop. I thought that there would be no loop by default, but it seems that it loops the queue by default and the second option is to just loop one of the songs.

### Suggested solution

This should be easy to implement probably. It's a very very basic feature of a lot of audio players.

**Comment by KRTirtho** [comment]
There is already a loop button :loop: 
![image](https://user-images.githubusercontent.com/61944859/224956472-9059d0b7-dd03-4953-9e5d-cf43093cc44d.png)

**Comment by isle9** [comment]
I think you misunderstood me. I know that there is an loop option already. I want an option to disable it.

**Reply by KRTirtho** [reply]
It's by default "Loop playlist" and is "Repeat One" when toggled on. So you want the playback to stop completely when the last track of the playlist finishes?

**Reply by isle9** [reply]
Exactly. That's what I thought would happen since that's how it usually works in apps. You either have no loop, loop one song or loop playlist/queue.
